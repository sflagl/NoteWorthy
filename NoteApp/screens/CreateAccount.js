import React, { Component } from 'react';
import { View, Text, StyleSheet} from 'react-native';
import { Col, Row, Grid } from "react-native-easy-grid";
import { Container, Header, Left, Body, Right, Button, Title, Content } from 'native-base';
import ButtonOutline from '../comps/button-outline';

import CreateCred from '../comps/create-account-cred';

import Icon from 'react-native-ionicons';


export class CreateAccount extends Component {
  static navigationOptions = {
    header: null
  };

  render() {

    return (
      
      <Container>
  
      <Content>
        <Grid>
          <Row size={2}>
            <Col style={{ padding: 10 }}>
              <CreateCred/>
            </Col>
  
          </Row>
          <Row size={1}>
            <Col style={{ padding: 10 }}>
              
              <ButtonOutline press={() => this.props.navigation.navigate('StartViewScreen')} title="Submit"/>   
            </Col>
          </Row>

        </Grid>
        </Content>
    </Container>
  
  
    
    )
  }
}

export default CreateAccount;

