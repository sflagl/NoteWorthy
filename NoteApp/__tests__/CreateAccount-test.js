import 'react-native';
import React from 'react';
import CreateAccount from '../screens/CreateAccount'

import renderer from 'react-test-renderer';

it('renders correctly', () => {
    const buttonTest = renderer.create(
        <CreateAccount />
    ).toJSON();
    expect(buttonTest).toMatchSnapshot();
});