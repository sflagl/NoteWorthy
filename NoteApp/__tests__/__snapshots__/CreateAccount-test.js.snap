// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "height": 1334,
    }
  }
  virtual={undefined}
>
  <RCTScrollView
    automaticallyAdjustContentInsets={false}
    contentContainerStyle={
      Array [
        Object {
          "padding": undefined,
        },
        undefined,
      ]
    }
    contentInset={
      Object {
        "bottom": 0,
      }
    }
    enableAutomaticScroll={true}
    enableResetScrollToCoords={true}
    extraHeight={75}
    extraScrollHeight={0}
    getScrollResponder={[Function]}
    handleOnScroll={[Function]}
    keyboardDismissMode="interactive"
    keyboardOpeningTime={250}
    keyboardShouldPersistTaps="handled"
    keyboardSpace={0}
    onScroll={[Function]}
    resetKeyboardSpace={[Function]}
    resetScrollToCoords={
      Object {
        "x": 0,
        "y": 0,
      }
    }
    scrollEventThrottle={1}
    scrollForExtraHeightOnAndroid={[Function]}
    scrollToEnd={[Function]}
    scrollToFocusedInput={[Function]}
    scrollToPosition={[Function]}
    showsVerticalScrollIndicator={true}
    style={
      Object {
        "backgroundColor": "transparent",
        "flex": 1,
      }
    }
    viewIsInsideTabBar={false}
    virtual={undefined}
  >
    <View>
      <View
        style={
          Object {
            "flex": 1,
            "flexDirection": "column",
          }
        }
      >
        <View
          size={2}
          style={
            Object {
              "flex": 2,
              "flexDirection": "row",
            }
          }
        >
          <View
            style={
              Object {
                "flex": 1,
                "flexDirection": "column",
                "padding": 10,
              }
            }
          >
            <View
              style={Object {}}
              virtual={undefined}
            >
              <View
                accessibilityComponentType={undefined}
                accessibilityLabel={undefined}
                accessibilityTraits={undefined}
                accessible={true}
                collapsable={undefined}
                hasTVPreferredFocus={undefined}
                hitSlop={undefined}
                isTVSelectable={true}
                nativeID={undefined}
                onLayout={undefined}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  Object {
                    "alignItems": "center",
                    "backgroundColor": "transparent",
                    "borderBottomWidth": 1,
                    "borderColor": "#D9D5DC",
                    "borderLeftWidth": 0,
                    "borderRightWidth": 0,
                    "borderTopWidth": 0,
                    "flexDirection": "row",
                    "marginLeft": 15,
                    "opacity": 1,
                  }
                }
                testID={undefined}
                tvParallaxProperties={undefined}
              >
                <TextInput
                  allowFontScaling={true}
                  editable={true}
                  placeholder="Full Name"
                  placeholderTextColor="#575757"
                  style={
                    Object {
                      "color": "#000",
                      "flex": 1,
                      "fontSize": 17,
                      "height": 50,
                      "lineHeight": 24,
                      "paddingLeft": 5,
                      "paddingRight": 5,
                      "top": 1.5,
                    }
                  }
                  underlineColorAndroid="rgba(0,0,0,0)"
                  virtual={undefined}
                />
              </View>
              <View
                accessibilityComponentType={undefined}
                accessibilityLabel={undefined}
                accessibilityTraits={undefined}
                accessible={true}
                collapsable={undefined}
                hasTVPreferredFocus={undefined}
                hitSlop={undefined}
                isTVSelectable={true}
                nativeID={undefined}
                onLayout={undefined}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  Object {
                    "alignItems": "center",
                    "backgroundColor": "transparent",
                    "borderBottomWidth": 1,
                    "borderColor": "#D9D5DC",
                    "borderLeftWidth": 0,
                    "borderRightWidth": 0,
                    "borderTopWidth": 0,
                    "flexDirection": "row",
                    "marginLeft": 15,
                    "opacity": 1,
                  }
                }
                testID={undefined}
                tvParallaxProperties={undefined}
              >
                <TextInput
                  allowFontScaling={true}
                  editable={true}
                  placeholder="Email"
                  placeholderTextColor="#575757"
                  style={
                    Object {
                      "color": "#000",
                      "flex": 1,
                      "fontSize": 17,
                      "height": 50,
                      "lineHeight": 24,
                      "paddingLeft": 5,
                      "paddingRight": 5,
                      "top": 1.5,
                    }
                  }
                  underlineColorAndroid="rgba(0,0,0,0)"
                  virtual={undefined}
                />
              </View>
              <View
                accessibilityComponentType={undefined}
                accessibilityLabel={undefined}
                accessibilityTraits={undefined}
                accessible={true}
                collapsable={undefined}
                hasTVPreferredFocus={undefined}
                hitSlop={undefined}
                isTVSelectable={true}
                nativeID={undefined}
                onLayout={undefined}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  Object {
                    "alignItems": "center",
                    "backgroundColor": "transparent",
                    "borderBottomWidth": 1,
                    "borderColor": "#D9D5DC",
                    "borderLeftWidth": 0,
                    "borderRightWidth": 0,
                    "borderTopWidth": 0,
                    "flexDirection": "row",
                    "marginLeft": 15,
                    "opacity": 1,
                  }
                }
                testID={undefined}
                tvParallaxProperties={undefined}
              >
                <TextInput
                  allowFontScaling={true}
                  editable={true}
                  placeholder="Password"
                  placeholderTextColor="#575757"
                  style={
                    Object {
                      "color": "#000",
                      "flex": 1,
                      "fontSize": 17,
                      "height": 50,
                      "lineHeight": 24,
                      "paddingLeft": 5,
                      "paddingRight": 5,
                      "top": 1.5,
                    }
                  }
                  underlineColorAndroid="rgba(0,0,0,0)"
                  virtual={undefined}
                />
              </View>
              <View
                accessibilityComponentType={undefined}
                accessibilityLabel={undefined}
                accessibilityTraits={undefined}
                accessible={true}
                collapsable={undefined}
                hasTVPreferredFocus={undefined}
                hitSlop={undefined}
                isTVSelectable={true}
                nativeID={undefined}
                onLayout={undefined}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  Object {
                    "alignItems": "center",
                    "backgroundColor": "transparent",
                    "borderBottomWidth": 1,
                    "borderColor": "#D9D5DC",
                    "borderLeftWidth": 0,
                    "borderRightWidth": 0,
                    "borderTopWidth": 0,
                    "flexDirection": "row",
                    "marginLeft": 15,
                    "opacity": 1,
                  }
                }
                testID={undefined}
                tvParallaxProperties={undefined}
              >
                <TextInput
                  allowFontScaling={true}
                  editable={true}
                  placeholder="Confirm Password"
                  placeholderTextColor="#575757"
                  style={
                    Object {
                      "color": "#000",
                      "flex": 1,
                      "fontSize": 17,
                      "height": 50,
                      "lineHeight": 24,
                      "paddingLeft": 5,
                      "paddingRight": 5,
                      "top": 1.5,
                    }
                  }
                  underlineColorAndroid="rgba(0,0,0,0)"
                  virtual={undefined}
                />
              </View>
            </View>
          </View>
        </View>
        <View
          size={1}
          style={
            Object {
              "flex": 1,
              "flexDirection": "row",
            }
          }
        >
          <View
            style={
              Object {
                "flex": 1,
                "flexDirection": "column",
                "padding": 10,
              }
            }
          >
            <View
              accessibilityComponentType={undefined}
              accessibilityLabel={undefined}
              accessibilityTraits={undefined}
              accessible={true}
              collapsable={undefined}
              hasTVPreferredFocus={undefined}
              hitSlop={undefined}
              isTVSelectable={true}
              nativeID={undefined}
              onLayout={undefined}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "alignItems": "center",
                  "alignSelf": "stretch",
                  "backgroundColor": "transparent",
                  "borderBottomWidth": 1,
                  "borderColor": "#5cb85c",
                  "borderLeftWidth": 1,
                  "borderRadius": 57,
                  "borderRightWidth": 1,
                  "borderTopWidth": 1,
                  "borderWidth": 1,
                  "color": "red",
                  "elevation": null,
                  "flexDirection": "row",
                  "height": 45,
                  "justifyContent": "center",
                  "margin": 10,
                  "opacity": 1,
                  "paddingBottom": 6,
                  "paddingTop": 6,
                  "shadowColor": null,
                  "shadowOffset": null,
                  "shadowOpacity": null,
                  "shadowRadius": null,
                }
              }
              testID={undefined}
              tvParallaxProperties={undefined}
            />
          </View>
        </View>
      </View>
    </View>
  </RCTScrollView>
</View>
`;
